packages/ag-charts-community/src/chart/data/dataModel.ts

```diff
@@ -322,7 +312,7 @@ export class DataModel<
         searchId: string,
         type: PropertyDefinition<any>['type'] = 'value'
     ): { type: typeof type; index: number; def: PropertyDefinition<any> } | never {
-        const { index, def } = this.resolveProcessedDataDefById(scope, searchId, type) ?? {};
```

Comment:
Can we drop `type` as a parameter to this method? If it was being set to anything other than `'value'` is this change introducing a bug?

(My recollection was we discussed this offline, so perhaps we just need to remove the `type` parameter above?)